workspace/scm.py

```diff
@@ -195,6 +195,12 @@ def merge_branch(branch, commit=None, squash=False, strategy=None):
         cmd.append('--strategy=' + strategy)
     if commit:
         cmd.append(commit)
+
```

Comment:
lets consolidate this condition and the one we already have for strategy above on line 194,